src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -21,20 +21,23 @@
     private final boolean isAlreadyPushed = false;
     private int ballsCollected;
 
-    public IntakeSubsystem(WPI_VictorSPX upperIntakeMotor,
-                           CANSparkMax bottomIntakeMotor,
-                           DigitalInput intakeSwitch,
-                           DoubleSolenoid intakeSolenoid) {
+    public IntakeSubsystem(
+            WPI_VictorSPX upperIntakeMotor,
+            CANSparkMax bottomIntakeMotor,
+            DigitalInput intakeSwitch,
+            DoubleSolenoid intakeSolenoid) {
         this.upperIntakeMotor = upperIntakeMotor;
         this.bottomIntakeMotor = bottomIntakeMotor;
         this.intakeSwitch = intakeSwitch;
         this.intakeSolenoid = intakeSolenoid;
         // upperIntakeMotor.setInverted(InvertType.OpposeMaster);
         intakeSwitchButton = new Trigger(intakeSwitch::get);
-        intakeSwitchButton.whenActive(new InstantCommand(() -> {
-            ballsCollected++;
-            logger.info("Ball count: " + ballsCollected);
-        }));
+        intakeSwitchButton.whenActive(
+                new InstantCommand(
+                        () -> {
+                            ballsCollected++;
+                            logger.info("Ball count: " + ballsCollected);
+                        }));
```

Comment:
Somethings up with your formatting settings because this is definitely not better than what was there before.